src/app/(auth)/login/page.tsx

```diff
@@ -1,5 +1,105 @@
+"use client";
+
 import type { FC } from "react";
+import useLoginForm from "@/hooks/useLoginForm";
+import Image from "next/image";
+import { Mail, Lock } from "lucide-react";
+import { Input } from "@/components/ui/input";
+import { Button } from "@/components/ui/button";
+import tabletImage from "@/public/svg/fragments/tablet-login.svg";
+import BgAuth from "@/components/login/background/bg-auth";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/components/ui/form";
 
-export const LoginPage: FC = () => {
-  return <div>LoginPage</div>;
+const LoginPage: FC = () => {
+  const { form, onSubmit, loginError, setLoginError } = useLoginForm();
+  return (
+    <main className="flex h-screen  justify-center relative">
+      <BgAuth />
+      <section className="w-full mx-5 md:mx-8 h-fit md:h-5/6 overflow-hidden rounded-3xl bg-white shadow-2xl self-center">
+        <div className="flex flex-col md:flex-row h-full">
+          <div className=" items-center h-full justify-center bg-gray-200 md:w-1/2 hidden sm:flex">
+            <Image src={tabletImage} alt="tablet login" className="w-3/5" />
+          </div>
+          <div className="p-5 md:p-12 md:w-1/2 my-auto">
+            <h2 className="my-0 md:my-6 text-4xl font-bold text-gray-800 md:text-start text-center [text-shadow:_0_3px_0_rgb(0_0_0_/_20%)]">
+              User login
+            </h2>
+            <Form {...form}>
+              <form
+                onSubmit={form.handleSubmit(onSubmit)}
+                className="space-y-6"
```

Comment:
Turn this back to `space-y-4`. The previous request is to add top gap just for the button. The alternative is by wrapping the submt button with a div and add `pt-6` to it